### PR TITLE
Fix typo in processor_stats.py

### PR DESCRIPTION
**Pull request by dtrai2**
For some processors there are no rule based statistics (such as
the clusterer or the selective_extractor). The latter one wasn't written
in snake case. To reduce the risk of multiple typos those processors 
were gathered in an exclusion list at the top of the class.

### CODE REVIEW

**Review by ppcad — APPROVED** [review]

**Review by ekneg54 — CHANGES_REQUESTED** [review]
please add a testcase for your described scenario

**Comment by dtrai2** [comment]
> please add a testcase for your described scenario

I revised the problem, fixed it differently and added a test case for it.

File: tests/unit/util/test_processor_stats.py [hunk 1]
```diff
@@ -213,3 +214,48 @@ def test_log_to_prometheus_exports_calls_gauge_labels(self, mock_labels):
...
+        )
+        clusterer.ps.aggr_data["processed"] = 78
+
+        self.status_tracker._pipeline.append(clusterer)
+
+        process_data = {}
+        self.status_tracker._add_per_processor_data(process_data=process_data)
+
+        assert process_data.get("Dropper1", {}).get("matches") == 0
+        assert process_data.get("Dropper1", {}).get("processed") == 13
```
Comment (ekneg54) [inline tests/unit/util/test_processor_stats.py:240]: Consider using the initialized value from dropper1

File: tests/unit/util/test_processor_stats.py [hunk 2]
```diff
@@ -213,3 +214,48 @@ def test_log_to_prometheus_exports_calls_gauge_labels(self, mock_labels):
...
+        assert all(
```
Comment (ekneg54) [inline tests/unit/util/test_processor_stats.py:241]: document assertion failure

**Review by ekneg54 — CHANGES_REQUESTED** [review]

**Review by urdit52 — APPROVED** [review]

File: tests/unit/util/test_processor_stats.py (hunk 1 above)
Comment (dtrai2) [reply tests/unit/util/test_processor_stats.py]: done

File: tests/unit/util/test_processor_stats.py (hunk 2 above)
Comment (dtrai2) [reply tests/unit/util/test_processor_stats.py:249]: done

**Review by ekneg54 — APPROVED** [review]

**Pull request merged**